subprojects/docs/src/docs/userguide/jdepend_plugin.adoc

```diff
@@ -13,84 +13,72 @@
 // limitations under the License.
 
 [[jdepend_plugin]]
-== The JDepend Plugin
+= The JDepend Plugin
 
 The JDepend plugin performs quality checks on your project's source files using http://clarkware.com/software/JDepend.html[JDepend] and generates reports from these checks.
 
 
 [[sec:jdepend_usage]]
-=== Usage
+== Usage
 
 To use the JDepend plugin, include the following in your build script:
 
-++++
-<sample id="useJDependPlugin" dir="codeQuality" title="Using the JDepend plugin">
-            <sourcefile file="build.gradle" snippet="use-jdepend-plugin"/>
-        </sample>
-++++
+=== Example: Using the JDepend plugin
+
+[source.multi-language-sample,groovy]
+.build.gradle
+----
+include::{samplesPath}/codeQuality/build.gradle[tag=use-jdepend-plugin]
+----
 
 The plugin adds a number of tasks to the project that perform the quality checks. You can execute the checks by running `gradle check`.
 
 Note that JDepend will run with the same Java version used to run Gradle.
 
 [[sec:jdepend_tasks]]
-=== Tasks
+== Tasks
```

Comment:
❌ Tables are only used when there is only one entry.

Reply:
Made into lists

Reply:
👍